<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>options</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>471</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>MusicPlayer</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxMPDisableSuspend">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Suspend</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxMPDisableLightOff">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Light Off</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>RadioPlayer</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxRPDisableSuspend">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Suspend</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxRPDisableLightOff">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Light Off</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>VideoPlayer</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxVPDisableSuspend">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Suspend</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxVPDisableLightOff">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Light Off</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>YoutubePlayer</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxYPDisableSuspend">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Suspend</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxYPDisableLightOff">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Light Off</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
